### PR TITLE
removes outdated browser prefixes

### PR DESCRIPTION
**Pull request by ghost**
on many css property's with user-select as an exception, there is no need for the browser prefix.
remove them will save some filespace

### CODE REVIEW

**Comment by williamdes** [comment]
All commits must have signoff :)

**Comment by BO41** [comment]
now, finally!

please review my changes. These are my first PR's to this project.

Please tell me if such changes are welcome. Or where I can find easy issues to solve :)

**Comment by Piyush3079** [comment]
The `-moz` and `webkit` prefixes are supported in latest browses only. Removing these may create some styling issues in some of the old `Firefox` and `Safari` browser.
Are you sure about removing these?

**Comment by BO41** [comment]
As far as I know, and I might be mistaken here, the prefixes allow the use of some propertys in earlier browser version. Or in some examples like [`user-select`](https://www.w3schools.com/csSref/css3_pr_user-select.asp) only cross browser with the prefix.
`user-select | 54.0 6.0-webkit- | 10.0-ms- | 2.0-moz- | 3.1-webkit- | 41.0 15.0-webkit-
--`

but other propertys like [`border-radius`](https://www.w3schools.com/cssref/css3_pr_border-radius.asp) don't really need the prefixes for ANY relative new browser
`border-radius | 5.0 4.0-webkit- | 9.0 | 4.0 3.0-moz- | 5.0 3.1-webkit- | 10.5
--`

of course there is a better support with the prefixes, but does it really make a difference if you support chrome 4 instead of 5 while the current version is 58 or something?
I don't think `html5` is supported back then.


But, I don't want to force anything, or even break any style. I just want to propose those changes and let you decide what is right :)
(sry, wasn't able to upload screenshots :/)

**Comment by williamdes** [comment]
TLDR;
https://caniuse.com

**Comment by BO41** [comment]
btw, not everywhere are the same prefixes used.
so if you decide to not remove them, we should use everywhere the same prefixes

File: themes/metro/jquery/jquery-ui.css [hunk 1]
```diff
@@ -5,945 +5,1109 @@
 * Copyright jQuery Foundation and other contributors; Licensed MIT */
```
Comment (Piyush3079) [inline themes/metro/jquery/jquery-ui.css]: Just a small suggestion:
This file is a vendor file downloaded from jquery.com and making code styling changes in vendor files may not be a good approach as we have lots of javascript and php vendor files which are not to the coding standards of PMA.
What's your suggestion regarding this? @williamdes

File: themes/original/jquery/jquery-ui.css [hunk 2]
```diff
@@ -5,945 +5,1109 @@
 * Copyright jQuery Foundation and other contributors; Licensed MIT */
```
Comment (Piyush3079) [inline themes/original/jquery/jquery-ui.css]: See above comment regarding this file

File: themes/pmahomme/jquery/jquery-ui.css [hunk 3]
```diff
@@ -5,945 +5,1109 @@
 * Copyright jQuery Foundation and other contributors; Licensed MIT */
```
Comment (Piyush3079) [inline themes/pmahomme/jquery/jquery-ui.css]: See above comment regarding this file

File: themes/metro/jquery/jquery-ui.css (hunk 1 above)
Comment (williamdes) [reply themes/metro/jquery/jquery-ui.css]: @Piyush3079 I think vendor files must not be changed, since they contain a version code.
They must be left intact.
Do we have a script that checks CS in vendor files ?

File: themes/original/jquery/jquery-ui.css (hunk 2 above)
Comment (BO41) [reply themes/original/jquery/jquery-ui.css]: ok, now I know. will remove those changes :)

File: themes/metro/jquery/jquery-ui.css (hunk 1 above)
Comment (Piyush3079) [reply themes/metro/jquery/jquery-ui.css]: I don't think so we have any such script.
Is it really required?

Comment (williamdes) [reply themes/metro/jquery/jquery-ui.css]: @Piyush3079 No, just asking :)
It would have explained why the files where prettified

**Comment by Piyush3079** [comment]
@BO41 Your signoff is by the name of `Lukas Walter` but DCO was expecting `BOB41`.
I think you should check you git configuration or rebase your commits with correct signoff :)

**Comment by BO41** [comment]
@Piyush3079 yeah thanks. I am trying to fix it all.
don't know what the problem of DCO is. I was asked to sign with my real name instead of my account name.
But I will figure it out, eventually

**Comment by BO41** [comment]
replaced with #14522 because this PR is broken.
